doc/BUILD.rst

```diff
@@ -39,7 +58,6 @@ Required
 Optional
 ''''''''
 
--  OpenSSL (for a better SHA-256 implementation; used in shader cache)
```

Comment:
I didn't mean to have this line removed completely. We still use sha256, and OpenSSL (or LibreSSL) can still be used to provide a better implementation. It's just that the "used in shader cache" part isn't worth mentioning.